### ISSUE TITLE
Model Analysis

### ISSUE THREAD

**Issue by BhavyaLight** [BhavyaLight/kaggle-predicting-Red-Hat-Business-Value#8]
Testing out different models 
Use softmax for everything :-
- [ ] KNN  @BhavyaLight
- [ ] Gaussian Naive bayes  @aakashgupta0205 
- [ ] Bernouli Naive Bayes  @aakashgupta0205 
- [ ] Multinomial Bayes @aakashgupta0205 
- [ ] logistic regression @sanjanajayakumar 
- [ ] Random forest   @aakashgupta0205 
- [ ] Xgboost @Yusuf007 
- [ ] SVM @Yusuf007 
---------------------------------------
- [ ] Ensemble learning, voting classfier ---------> together
- [ ] ANN @sanjanajayakumar 
- [ ] DNN @BhavyaLight 
/...to add more
